sounds/views.py

```diff
@@ -125,10 +127,18 @@ def remixed(request):
 
 
 def random(request):
-    sound_id = Sound.objects.random()
-    if sound_id is None:
+    sounds = get_random_sounds_from_solr(1)
```

Comment:
Will this method be used in any other places? I wonder if we don't need to get multiple random sounds - what if this method only returns 1 sound?